indexer/services/comlink/src/controllers/api/v4/addresses-controller.ts

```diff
@@ -294,6 +306,72 @@ class AddressesController extends Controller {
       childSubaccounts: subaccountResponses,
     };
   }
+
+  @Post('/:address/registerToken')
+  public async registerToken(
+    @Path() address: string,
+      @Body() body: { token: string, language: string },
+  ): Promise<void> {
+    const { token, language } = body;
+    const foundAddress = await WalletTable.findById(address);
+    if (!foundAddress) {
+      throw new NotFoundError(`No address found with address: ${address}`);
+    }
+    try {
+      const allTokens = await FirebaseNotificationTokenTable.findAll(
+        { address },
+        [],
+      );
+      // Delete tokens older than a month
+      const oneMonthAgo = new Date();
```

Comment:
Do we want to do this as part of the request path? Seems more suited for a roundtable task.